discord-websockets.c

```diff
@@ -99,8 +99,9 @@ ws_send_resume(struct discord_ws_s *ws)
   char fmt_payload[] = \
     "{\"op\":6,\"d\":{\"token\":\"%s\",\"session_id\":\"%s\",\"seq\":%d}}";
   char payload[MAX_PAYLOAD_LEN];
-  snprintf(payload, sizeof(payload)-1, fmt_payload,
+  int ret = snprintf(payload, MAX_PAYLOAD_LEN, fmt_payload,
       ws->p_client->settings.token, ws->session_id, ws->payload.seq_number);
+  ASSERT_S(ret < MAX_PAYLOAD_LEN, "Out of bounds write attempt");
 
```

Comment:
I liked your use of `sizeof(payload)`   it's more resilient to change.

Reply:
Thats true, I will revert it then. I did this to avoid compiler warnings for `ret` being a different type than `sizeof(payload)`, but a simple casting will accomplish the same thing.